packages/amplify-category-api/src/__tests__/graphql-transformer/transform-graphql-schema-v2.test.ts

```diff
@@ -1,7 +1,7 @@
 import { $TSContext, pathManager, ApiCategoryFacade } from '@aws-amplify/amplify-cli-core';
 import { printer } from '@aws-amplify/amplify-prompts';
 import { constructTransformerChain } from '@aws-amplify/graphql-transformer';
-import { constructDataSourceMap, DDB_DEFAULT_DATASOURCE_TYPE } from '@aws-amplify/graphql-transformer-core';
+import { DataSourceType, DynamoDBProvisionStrategy } from 'graphql-transformer-core';
```

Comment:
We still have the `DataSourceType` and it's `*Strategy` counter parts?

Reply:
Yes, I added multiple notes in the declaration about why we're keeping it